### PR TITLE
JAV-129 supporting both cse & servicecomb in config properties

### PR DESCRIPTION
**Pull request by zhengyangyong**

### CODE REVIEW

File: foundations/foundation-config/src/main/java/io/servicecomb/config/ConfigUtil.java [hunk 1]
```diff
@@ -169,4 +201,45 @@ public static void installDynamicConfig() {
...
+  private static class ServiceCombPropertyUpdateListener implements WatchedUpdateListener {
+
+    private final ConcurrentMapConfiguration injectConfig;
+
+    ServiceCombPropertyUpdateListener(ConcurrentMapConfiguration injectConfig) {
+      this.injectConfig = injectConfig;
+    }
+
+    @Override
+    public void updateConfiguration(WatchedUpdateResult watchedUpdateResult) {
```
Comment (wujimin) [inline foundations/foundation-config/src/main/java/io/servicecomb/config/ConfigUtil.java]: i'm not sure, if "cse" will part of the config key not just the prefix, like: abc.cse.123 or abc.csexxx.123
maybe just change "cse." prefix to "servicecomb." is better?

File: foundations/foundation-config/src/test/java/io/servicecomb/config/TestConfigurationSpringInitializer.java [hunk 2]
```diff
@@ -32,4 +36,11 @@ public void testAll() {
         Assert.assertEquals(3, listO.size());
         Assert.assertEquals(null, ConfigUtil.getProperty("notExist"));
     }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
```
Comment (wujimin) [inline foundations/foundation-config/src/test/java/io/servicecomb/config/TestConfigurationSpringInitializer.java]: full config teardown is io.servicecomb.serviceregistry.ArchaiusUtils.resetConfig()
if we have a test common project, can move ArchaiusUtils to it.

File: foundations/foundation-config/src/test/resources/microservice.yaml [hunk 3]
```diff
@@ -42,4 +42,11 @@ cse:
 
 service_description:
   name: testDemo
+
+
+servicecomb:
```
Comment (zhengyangyong) [inline foundations/foundation-config/src/test/resources/microservice.yaml]: 模拟yaml存在servicecomb配置项，duplicate一份cse,应该使用servicecomb.whatever.property之类的会比较好，tracing有实际的使用场景，这里误用了

File: foundations/foundation-config/src/main/java/io/servicecomb/config/ConfigUtil.java (hunk 1 above)
Comment (zhengyangyong) [inline foundations/foundation-config/src/main/java/io/servicecomb/config/ConfigUtil.java]: Done

File: foundations/foundation-config/src/test/java/io/servicecomb/config/TestConfigurationSpringInitializer.java (hunk 2 above)
Comment (zhengyangyong) [inline foundations/foundation-config/src/test/java/io/servicecomb/config/TestConfigurationSpringInitializer.java]: 应该需要一次对demo,integration-tests和samples的整体整理,可以考虑在JAV-108这样的过程中一起处理？

File: foundations/foundation-config/src/main/java/io/servicecomb/config/ConfigUtil.java (hunk 1 above)
Comment (zhengyangyong) [inline foundations/foundation-config/src/main/java/io/servicecomb/config/ConfigUtil.java]: will change all test case config property key to "servicecomb.cse.servicecomb.xxx"

File: foundations/foundation-config/src/test/resources/microservice.yaml [hunk 4]
```diff
@@ -42,4 +42,10 @@ cse:
...
+  cse:
+    servicecomb:
+      file: vaule
```
Comment (seanyinx) [inline foundations/foundation-config/src/test/resources/microservice.yaml]: spelling error

File: foundations/foundation-config/src/test/java/io/servicecomb/config/TestConfigurationSpringInitializer.java (hunk 2 above)
Comment (wujimin) [inline foundations/foundation-config/src/test/java/io/servicecomb/config/TestConfigurationSpringInitializer.java]: infact, i feel that move ArchaiusUtils to src/main/java of foundation-config is not so bad
but seanyinx has different idea

File: foundations/foundation-config/src/test/resources/microservice.yaml (hunk 4 above)
Comment (zhengyangyong) [inline foundations/foundation-config/src/test/resources/microservice.yaml]: Done

File: foundations/foundation-config/src/test/java/io/servicecomb/config/TestConfigurationSpringInitializer.java [hunk 5]
```diff
@@ -32,4 +36,11 @@ public void testAll() {
         Assert.assertEquals(3, listO.size());
         Assert.assertEquals(null, ConfigUtil.getProperty("notExist"));
     }
+
+//    @AfterClass
```
Comment (zhengyangyong) [inline foundations/foundation-config/src/test/java/io/servicecomb/config/TestConfigurationSpringInitializer.java]: Done

**Pull request merged**